watch_faces/BinarySystem/source/DataViews/SunView.mc

```diff
@@ -0,0 +1,62 @@
+using Toybox.Graphics as Gfx;
+using Toybox.Application as App;
+using Toybox.Time as Time;
+using Toybox.WatchUi as Ui;
+using Toybox.System as Sys;
+using Toybox.Activity as Activity;
+
+
+class SunView extends Ui.Drawable
+{
+    var showSun = false;
+    function initialize(params)
+        {
+            Drawable.initialize(params);
+            var x = params.get(:x);
+            var y = params.get(:y);
+            showSun = params.get(:showSun);
```

Comment:
there is no real reason to have the showsun here. if this is an opplicationoption it makes sense. otherwise this can simply be achived with removing the drawable from the layout